### ISSUE TITLE
Documentation of 1executor.update_parameters` arguments

### ISSUE THREAD

**Issue by JoeZiminski** [facebookincubator/submitit#1735]
Thanks for making this excellent tool! It works basically out of the books and has proved extremely useful.

Apologies if I am missing something obvious, but it would be great  to have a full list of the arguments `executor.update_parameters(timeout_min=1, slurm_partition="dev")` accepts. I always end up flailing about looking through examples and source code until I find the name of a parameter I want to pass / if it is supported. Maybe there is a better way I'm missing!

**Comment by yinkaai** [comment]
I agree very much, although I feel that the keywords here are the keyword list of slurm([https://slurm.schedmd.com/sbatch.html]), but I often encounter warnings that I am not using the keywords properly, and I don't know if it is triggered by slurm or submitit. I hope the submitit devs can add a little clarification in the README.
